pandas/formats/templates/html.tpl

```diff
@@ -0,0 +1,61 @@
+{%- block before_style -%}{%- endblock before_style -%}
+{% block style %}
+<style  type="text/css" >
+{% block table_styles %}
+{% for s in table_styles %}
+    #T_{{uuid}} {{s.selector}} {
+    {% for p,val in s.props %}
+      {{p}}: {{val}};
+    {% endfor -%}
+    }
+{%- endfor -%}
+{% endblock table_styles %}
+{% block before_cellstyle %}{% endblock before_cellstyle %}
+{% block cellstyle %}
+{%- for s in cellstyle %}
+    #T_{{uuid}}{{s.selector}} {
+    {% for p,val in s.props %}
+        {{p}}: {{val}};
+    {% endfor %}
+    }
+{%- endfor -%}
+{%- endblock cellstyle %}
+</style>
+{%- endblock style %}
+{%- block before_table %}{% endblock before_table %} 
+{%- block table %}  
+<table id="T_{{uuid}}" {% if table_attributes %}{{ table_attributes }}{% endif %}> 
+{%- block caption %} 
+{%- if caption -%} 
+    <caption>{{caption}}</caption> 
+{%- endif -%} 
+{%- endblock caption %} 
+{%- block thead %} 
+<thead> 
+    {%- for r in head %} 
+    <tr> 
+        {%- for c in r %} 
+        {%- if c.is_visible != False %} 
+        <{{ c.type }} class="{{c.class}}" {{ c.attributes|join(" ") }}>{{c.value}}</{{ c.type }}>
+        {%- endif %} 
+        {%- endfor %} 
+    </tr> 
+    {%- endfor %} 
+</thead> 
+{%- endblock thead %} 
+{%- block tbody %} 
+<tbody> 
+    {%- for r in body %} 
```

Comment:
Could you add a `rows` block here? (or alternatively empty `before_rows` and  `after_rows`).   What I was trying to do is add additional rows at the end of the table which I don't think is possible with these extension points.